ui/app/styles/utils/_size_variables.scss

```diff
@@ -3,18 +3,23 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
+@function rem($value, $base: 16) {
+  @return calc($value / $base) + 0rem;
+}
+
 /* General sizing in rem values used largely for text sizing.*/
-$size-1: 3rem; // 48px, same as $spacing-xxl
```

Comment:
I'm confused about this, because when I [convert](https://nekocalc.com/rem-to-px-converter) 3rem to px it's 48?

Reply:
Oh it's because changing the base size to 16px means that 1rem is now 16 instead of 14